Userland/Utilities/disasm.cpp

```diff
@@ -130,7 +130,26 @@ ErrorOr<int> serenity_main(Main::Arguments args)
             is_first_symbol = false;
         }
 
-        outln("{:p}  {}", virtual_offset, insn.value().to_string(virtual_offset, symbol_provider));
+        size_t length = insn.value().length();
+        StringBuilder builder;
+        builder.appendff("{:p}  ", virtual_offset);
+        for (size_t i = 0; i < 7; i++) {
+            if (i < length)
+                builder.appendff("{:02x} ", asm_data[offset + i]);
+            else
+                builder.append("   "sv);
+        }
+        builder.append(" "sv);
+        builder.append(insn.value().to_string(virtual_offset, symbol_provider));
+        outln("{}", builder.to_string());
```

Comment:
You can avoid a string allocation here by outputting `builder.string_view()` instead of calling `to_string()`.

cc @nico :^)